locales-pending/landing-premium.ftl

```diff
@@ -12,3 +12,69 @@ landing-premium-hero-image-removal-badge-label = exposures auto‑removed
 
 # This is a label underneath a big number "50" - it's an image that demos Monitor.
 landing-premium-hero-image-chart-label = fixed
+
+landing-premium-plans-heading = Choose your level of protection
+landing-premium-plans-lead = We believe in your right to privacy, so data breach monitoring is always free. For more robust protection, { -brand-monitor } also offers continuous automatic removal of your personal information.
+landing-premium-plans-table-annotation-plus = Recommended
+landing-premium-plans-table-heading-feature = Features
+# Users can click a button with this label to see a popover with a bit more detail.
+landing-premium-plans-table-feature-callout-trigger = More info
+landing-premium-plans-table-heading-free-title = { -brand-monitor }
+landing-premium-plans-table-heading-free-subtitle = Free breach alerts
+landing-premium-plans-table-heading-plus-title = { -brand-monitor } <b>Plus</b>
+landing-premium-plans-table-heading-plus-subtitle = Automatic data removal
+# Variables:
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+landing-premium-plans-table-feature-scan-label =
+    { $dataBrokerTotalCount ->
+        [one] Scan { $dataBrokerTotalCount } data broker site that may be selling your personal info
+       *[other] Scan { $dataBrokerTotalCount } data broker sites that may be selling your personal info
+    }
+landing-premium-plans-table-feature-scan-free = One-time
+landing-premium-plans-table-feature-scan-plus = Monthly
+landing-premium-plans-table-feature-removal-label = Remove personal info from sites that are selling it
+landing-premium-plans-table-feature-removal-free = Manual removal
+landing-premium-plans-table-feature-removal-plus = Automatic removal
+landing-premium-plans-table-feature-removal-free-callout = We’ll let you know which data brokers are selling your info so you can contact them to request removal.
+# Variables:
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+landing-premium-plans-table-feature-removal-plus-callout =
+    { $dataBrokerTotalCount ->
+        [one] We’ll automatically request removal of your private info across { $dataBrokerTotalCount } data broker site.
```

Comment:
Does this need a singular version since it will always be 190+?

Reply:
Yes, because the target is not much English but other locales (if we ever expose these strings for localization).